src/DependencyInjection/SonataTranslationExtension.php

```diff
@@ -102,6 +106,8 @@ public function load(array $configs, ContainerBuilder $container)
     /**
      * @param array $translationTargets
      *
+     * @phpstan-param array<string, array<string, class-string[]>> $translationTargets
```

Comment:
That variable is only ever used in a foreach, and the key is not used…
```suggestion
     * @phpstan-param iterable<string, array<string, class-string[]>> $translationTargets
```

or even
```suggestion
     * @phpstan-param iterable<array<string, class-string[]>> $translationTargets
```